internal/workflow/processing_test.go

```diff
@@ -162,16 +163,20 @@ func (s *ProcessingWorkflowTestSuite) TestPackageConfirmation() {
 		&activities.DownloadActivityResult{Path: "/tmp/enduro123456/" + key}, nil,
 	)
 
+	s.env.OnActivity(activities.UnarchiveActivityName, sessionCtx,
+		&activities.UnarchiveParams{SourcePath: "/tmp/enduro123456/" + key},
+	).Return(
+		&activities.UnarchiveResult{DestPath: "/tmp/enduro123456/extract"}, nil,
```

Comment:
It would be nice to have these two paths as variables and reuse them. That would have helped with some modifications I made in a fork where the path changed in a preprocessing step and I had to modify its usage in all calls later.